### DISCUSSION TITLE
How to turn on a GPU runtime

### DISCUSSION THREAD

**Discussion by Jiraxys** [neonbjb/tortoise-tts#302] [Q&A]
Hello,
i am fairly new to this, when using the program, it takes hours to generate speech because it uses 100% of my CPU and 0% of my GPU. how do i make it use my GPU instead as mentioned by the Author in https://github.com/neonbjb/tortoise-tts/blob/main/tortoise_tts.ipynb ?

EDIT:
NVM just found out yoy have to install pytorch CUDA version, but it still takes 25 minutes for a phrase despite using 3060 ti. it only uses 0.3% of the gpu and 100% cpu

**Comment by MackNcD** [comment]
yeah same, that makes no sense to me, how is it using multiple GBs of GPU in collab and basically nothing locally?

**Reply by JMcCoding** [reply]
Looks like the original author had the wrong version of pytorch installed, and it didn't recognize he had a GPU capable of being utilized.

**Reply by neonbjb** [reply]
lol no, you really think I could have built these things on CPU?

I think most people who have this issue are not following the readme in the correct order. I specifically state that you must install torch manually. Unfortunately if you get the ordering wrong and install Tortoise first, it will automatically install torch-cpu and the subsequent instructions will not undo that. Just python things..........

**Reply by JMcCoding** [reply]
I should have used a little better English.  I didn't mean the repo author, meant the post author, especially based on his edit.

**Edit**: Thanks for saying something so I could clarify.
